File: src/WellTool.Core/hutool-core/src/test/java/cn/hutool/core/util/IssueIBP6T1Test.java
package cn.hutool.core.util;

import org.junit.jupiter.api.Assertions;
import org.junit.jupiter.api.Test;

public class IssueIBP6T1Test {
	@SuppressWarnings("DataFlowIssue")
	@Test
	void isValidCard10Test(){
		Assertions.assertEquals("true", IdcardUtil.isValidCard10("1608214(1)")[2]);
		Assertions.assertEquals("true", IdcardUtil.isValidCard10("1608214（1）")[2]);
	}
}

File: src/WellTool.Core.Tests/IssueIBP6T1Test.cs
using Xunit;
using WellTool.Core;
using WellTool.Core.Util;

namespace WellTool.Core.Tests
{
    /// <summary>
    /// Issue #IBP6T1 测试
    /// </summary>
    public class IssueIBP6T1Test
    {
        [Fact]
        public void IsValidCard10Test()
        {
            Assert.Equal("true", IdcardUtil.IsValidCard10("1608214(1)")[2]);
            Assert.Equal("true", IdcardUtil.IsValidCard10("1608214（1）")[2]);
        }
    }
}
